client/console/src/main/java/org/apache/syncope/client/console/init/ClassPathScanImplementationLookup.java

```diff
@@ -51,6 +53,8 @@
     private List<Class<? extends BaseExtPage>> extPages;
 
     private List<Class<? extends BaseExtWidget>> extWidgets;
+    
+    private List<String> basePackages = Arrays.asList("org.apache.syncope.client.console");
```

Comment:
Prior to this change, one could create classes, in its own Maven-based Syncope project, under any package, now it is forced to do it under `org.apache.syncope.client.console` but we got more startup speed.
I guess it is an acceptable trade-off.

LGTM.